.gitlab/install_ddtrace_deps.rb

```diff
@@ -1,53 +1,79 @@
-require "open3"
-require "rubygems"
-require "bundler"
-require "fileutils"
-require "pathname"
+require 'open3'
+require 'rubygems'
+require 'bundler'
+require 'fileutils'
+require 'pathname'
 
 ruby_api_version = Gem.ruby_api_version
 
 current_path = Pathname.new(FileUtils.pwd)
 
-artifact_path = current_path.join("pkg")
+tmp_path = current_path.join('tmp')
 
-lock_file_path = artifact_path.join("Gemfile.lock")
-versioned_path = artifact_path.join(ruby_api_version)
+versioned_path = tmp_path.join(ruby_api_version)
+
+FileUtils.mkdir_p(versioned_path, verbose: true)
+
+gemfile_file_path = versioned_path.join('Gemfile')
+
+File.open(gemfile_file_path, 'w') do |file|
+  file.write("source \"https://rubygems.org\"\n")
+  file.write("gem 'ddtrace', '#{ENV.fetch('RUBY_PACKAGE_VERSION')}', path: '#{current_path}'")
+end
+
+$stdout.puts '=== Reading Gemfile ==='
```

Comment:
```suggestion
puts '=== Reading Gemfile ==='
```
Do we need `$stdout` declared explicitly for a particular reason?